GroversAlgorithm/GroversAlgorithm.ipynb

```diff
@@ -270,6 +306,15 @@
     "    You need to modify it to add the $-1$ phase to only the $|0...00\\rangle$ state instead.\n",
     "    <br/><br/>\n",
     "    Alternatively, you can use the same trick as in the oracle converter task.<br>\n",
+    "</details>\n",
+    "<br>\n",
+    "\n",
+    "<details>\n",
+    "  <summary><b>IMPORTANT : The test will pass but its not the correct implementation. For More details click here</b></summary>\n",
```

Comment:
Looks like this hint comes from the version of #505 before the latest version? Same comment as in #505, let's add this to hint 1. 

Also, I would make the information in hint 3 that you added in the latest version of that PR a lot shorter - the information about eigenvalues is not necessary and somewhat confusing, it's enough to point the learner to the R operation.

Reply:
Done